apps/nextjs/src/pages/index.tsx

```diff
@@ -28,6 +30,10 @@ export default function Home() {
         <HelenaGlowacki />
         <RamRaghavSharma />
         <MinhLe />
+        <div>
+          {isSignedIn ? "" : <SignInButton mode="modal" />}
+          <UserButton afterSignOutUrl="/" />
+        </div>
```

Comment:
is there a way to have the clerk signin page instead of the button?

Reply:
Yes if you don't explicitly pass in the mode it'll use a full page for signin, I can change it to that